README.md

```diff
@@ -271,9 +281,20 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 "state_timeout_in_seconds": 600,
 
                 // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account
+                // OIDC user to an iRODS user account.
+                // 'irods_user_claim' and 'user_attribute_mapping' cannot be used together
                 "irods_user_claim": "irods_username",
 
+                // The mapping of a user to the provided values. All values must
+                // be matched to map an OIDC user to an iRODS user account.
+                // 'irods_user_claim' and 'user_attribute_mapping' cannot be used together
```

Comment:
- Replace single quotes with double quotes for consistency
- Missing period

Reply:
Done.